pkg/lib/files/errors.go

```diff
@@ -130,3 +147,30 @@ func ErrorUnexpected() error {
 		message: "an unexpected error occurred",
 	}
 }
+func ErrorFileDoesNotExist(path string) error {
```

Comment:
Are all (or most) of these errors used only for wrapping, and if so should we convert them to strings?

Reply:
I don't think so, because we still need to build the string from `path`